spec/views/programs/show.html.erb_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe "programs/show", type: :view do
+  before(:each) do
+    @program = assign(:program, Program.create!(
+      :title => "Title",
```

Comment:
Indent the first parameter one step more than `Program.create!(`.<br>Use the new Ruby 1.9 hash syntax.